src/App.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import { FC, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [query, changeQuery] = useState('');
+  const visibleMovies = query
+    ? moviesFromServer.filter(movie => {
+      const inputValue = query
+        .toLowerCase()
+        .trim();
+
+      const title = movie.title
+        .toLowerCase()
+        .includes(inputValue);
+      const description = movie.description
+        .toLowerCase()
+        .includes(inputValue);
```

Comment:
implement filter by adding the ability to ignore extra spaces between entered words.
It would fun experience, i promise.